src/Components/InstanceCounter/index.js

```diff
@@ -37,18 +39,34 @@ const InstanceCounter = () => {
   };
 
   return (
-    <NumberInput
-      max={MAX_INSTANCES}
-      min={MIN_INSTANCES}
-      value={chosenNumOfInstances}
-      onMinus={onMinus}
-      onChange={onChange}
-      onPlus={onPlus}
-      inputName="instances"
-      inputAriaLabel="number of instances"
-      minusBtnAriaLabel="instances-minus"
-      plusBtnAriaLabel="instances-plus"
-    />
+    <>
+      {!(vCPUCount * chosenNumOfInstances <= 32) && (
```

Comment:
We could use a boolean hook. But this would have to be in the parent component, and we'd have to pass it to the instance type selector to get it updated after changing the instance count and after changing the count. 
Tbh, I do not know if it is a good manner to render components with such values without a hook, let me know. :)

Reply:
I guess it is not, but it will work, so I do not care much :stuck_out_tongue: 

Btw I love your condition! I'd do the negation `( i * j > 32 )` because of my min ops OCD, but yours is much more readable, :orange_heart: it.